mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/ErrorBarChart.cs

```diff
@@ -1,6 +1,6 @@
 //-------------------------------------------------------------
-// <copyright company=�Microsoft Corporation�>
-//   Copyright � Microsoft Corporation. All Rights Reserved.
+// <copyright company=�Microsoft Corporation�>
+//   Copyright � Microsoft Corporation. All Rights Reserved.
```

Comment:
This doesn't look right.  Please convert the file from Windows 1252 to UTF-8

Reply:
My mistake. I will fix them. Thank you for the review.

Reply:
That's weird. It's already UTF-8.

Reply:
![image](https://user-images.githubusercontent.com/1798510/73120352-23a19a80-3f6d-11ea-9435-c0a0ba3bf7b0.png)

Reply:
@lambdageek I made sure it is UTF-8. I replaced those weird characters with their previous values in order to fix it.